ford_fusion_sehybrid_2019/docker-compose.yml

```diff
@@ -108,6 +108,7 @@ services:
 
   velodyne_lidar_driver:
     image: usdotfhwastoldev/carma-velodyne-lidar-driver:develop
+    image: usdotfhwastol/carma-velodyne-lidar-driver:develop
```

Comment:
This duplicated line needs to be removed. (It also has the wrong organization name)

Reply:
Removed Duplicate lines and updated to correct Organization name